### DISCUSSION TITLE
404 error downloading Cost summary report

### DISCUSSION THREAD

**Discussion by lundejd** [microsoft/finops-toolkit#1175] [Q&A] [answered]
From this page https://microsoft.github.io/finops-toolkit/power-bi/cost-summary 

I get a 404 error when trying to download the .7 version of the PBIX.

**Comment by EQ-GALLI** [comment]
I dont get the 404 I do however get redirected to this project. you can still download the 0.6 reports using the following address:

https://github.com/microsoft/finops-toolkit/releases/download/v0.6/CostSummary.pbix

You just need to change the name of the report

**Reply by lundejd** [reply]
Yea, now it seems to just redirect back to itself. 

I'm looking for v0.7 though. thanks.

**Reply by EQ-GALLI** [reply]
Same, if you get a response please let me know ;-)

**Comment by lundejd** [comment, marked as answer]
The .pbit files are there inside a .zip file. PowerBI-storage.zip is the one I use, if you move to data explorer I believe its PowerBI-kql.zip

I was able to get what I needed from those files and no longer need the full .pbix
